dfx-library/visualstudio/dfxplugin-version.rc

```diff
@@ -21,6 +21,7 @@
 #endif
 
 #define PLUGIN_DISPLAY_NAME	PLUGIN_NAME_STRING
+#define PLUGIN_VERSION_STRING #PLUGIN_VERSION_MAJOR #PLUGIN_VERSION_MINOR #PLUGIN_VERSION_BUGFIX

```

Comment:
Indeed this doesn't compile (turns out mingw's windres just uses gcc to preprocess). Two problems: I think it is only actually legal to #stringify macro arguments, not already-existing defines. But if you try like #define CONCAT(a, b) #a "." #b and calling CONCAT(MAJOR, MINOR) then you just get a string like "MAJOR.MINOR". This page explains, but also the solution of using two macros:
https://gcc.gnu.org/onlinedocs/gcc-3.4.6/cpp/Argument-Prescan.html#Argument-Prescan

Reply:
Ah yes, I remember having to do this once upon a time for generating unique AU View class names in:
https://github.com/sophiapoirier/destroyfx/blob/8302de1fbd98a52125995d7bf6b4b835ecdacae7/dfxgui/dfxgui-auviewfactory.mm#L35
Thanks for the multiple fixes!